spin/dataformat-configuration-global/README.md

```diff
@@ -113,6 +113,5 @@ Where to place the data format configuration JAR on which application server:
 
 | Application Server           | Configuration                                                           |
 | ---------------------------- | ----------------------------------------------------------------------- |
-| Tomcat                       | Put the JAR and its transitive dependencies into `${TOMCAT_HOME}/lib`.  |
```

Comment:
Why was Tomcat removed here?

Reply:
The example does not work with Tomcat and is not easy to fix. The pom specifies the `camunda-spin-dataformat-json-jackson` dependency, but tomcat provides `camunda-spin-dataformat-all` which shades the previous one.

Reply:
Sorry, should have clarified before assigning to you.

Reply:
This makes sense. Let's keep the change 👍